### PR TITLE
Oren/user results service

### PR DESCRIPTION
**Pull request by orenor10**

### CODE REVIEW

File: src/components/userResults/module.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-dashboard.userResults', []);
```
Comment (assafshp) [inline src/components/userResults/module.js]: Doesn't ENV is needed here ? the service needs it

File: src/components/userResults/userResults.service.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+            userResultsService.getExerciseResults = function (uid) {
+                return getResultsFromFB(ENV.studentAppName + '/exerciseResults', uid);
+            };
+
+            userResultsService.getExamResults = function (uid) {
+                return getResultsFromFB(ENV.studentAppName + '/examResults', uid);
+            };
+
+            function getResultsFromFB(path, uid) {
```
Comment (assafshp) [inline src/components/userResults/userResults.service.js]: this function only return completed = true ?

File: src/components/userResults/module.js (hunk 1 above)
Comment (orenor10) [inline src/components/userResults/module.js]: ENV is injected in UserResultsService.
here is the module declaration

File: src/components/userResults/userResults.service.js (hunk 2 above)
Comment (orenor10) [inline src/components/userResults/userResults.service.js]: return array of completed exams/exercises

**Comment by assafshp** [comment]
jenkins-test

**Comment by assafshp** [comment]
jenkins-test

**Comment by assafshp** [comment]
jenkins-test

**Pull request merged**